spec/models/user_spec.rb

```diff
@@ -1675,4 +1675,41 @@ def it_should_not_send_survey
       expect(user.second_last_signed_in_at).to eq(event2.reload.created_at)
     end
   end
+
+  describe '#has_fed_or_mil_email' do
```

Comment:
Nit: This doesn't match the method name.

```suggestion
  describe '#has_fed_or_mil_email?' do
```